conda/recipes/librmm/recipe.yaml

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2018-2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  cuda_version: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[:2] | join(".") }}
+  cuda_major: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[0] }}
+  date_string: ${{ env.get("RAPIDS_DATE_STRING") }}
```

Comment:
I grabbed these from rapidsai/cugraph#4551 as currently `conda-recipe-manager` doesn't have support for handling converting from the extended jinja2 syntax to the subset that rattler supports